lib/shared/src/event/log_event/lua.rs

```diff
@@ -31,16 +32,22 @@ impl<'a> FromLua<'a> for LogEvent {
 #[cfg(test)]
 mod test {
     use super::*;
-    use crate::event::Event;
+    use rlua::Lua;
 
-    #[test]
+    #[test_env_log::test]
```

Comment:
What does this do?

Reply:
This is https://crates.io/crates/test-env-log, since we don't have `vector::test_util::trace_init()` this accomplishes similar.